src/dialogs/more-info/controls/more-info-input_datetime.ts

```diff
@@ -0,0 +1,101 @@
+import { HassEntity } from "home-assistant-js-websocket";
+import { css, CSSResultGroup, html, LitElement, TemplateResult } from "lit";
+import { customElement, property } from "lit/decorators";
+import "../../../components/ha-date-input";
+import "../../../components/ha-time-input";
+import { UNAVAILABLE_STATES, UNKNOWN } from "../../../data/entity";
+import { setInputDateTimeValue } from "../../../data/input_datetime";
+import type { HomeAssistant } from "../../../types";
+
+@customElement("more-info-input_datetime")
+class MoreInfoInputDatetime extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ attribute: false }) public stateObj?: HassEntity;
+
+  protected render(): TemplateResult {
+    if (!this.stateObj) {
+      return html``;
+    }
+
+    return html`
+        ${
+          this.stateObj.attributes.has_date
+            ? html`
+                <ha-date-input
+                  .value=${`${this.stateObj.attributes.year}-${this.stateObj.attributes.month}-${this.stateObj.attributes.day}`}
+                  .disabled=${UNAVAILABLE_STATES.includes(this.stateObj.state)}
+                  @value-changed=${this._dateChanged}
+                >
+                </ha-date-input>
+                ${this.stateObj.attributes.has_time ? "," : ""}
```

Comment:
```suggestion
                ${this.stateObj.attributes.has_time ? "," : ""}
```
Is this `,` desired? kinda looks strange, can we do space or some padding instead?

Reply:
It was copied from the entity row, but yeah, I kinda agree